cmd/kubectl-gadget/tcptop.go

```diff
@@ -0,0 +1,314 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/tcptop/types"
+)
+
+var (
+	nodeTCPStats map[string][]types.Stats
+	mtx          sync.Mutex
+	ended        bool = false
+)
+
+var (
+	// arguments
+	intervalOutput int = 1
+
+	// flags
+	maximumRows int
+	sortTypeStr string
+	sortType    types.SortBy
+	filteredPid uint
+	family      uint
+)
+
+func translateFamily(family uint) (string, error) {
+	switch family {
+	case 4:
+		return strconv.Itoa(syscall.AF_INET), nil
+	case 6:
+		return strconv.Itoa(syscall.AF_INET6), nil
```

Comment:
> I think this should be done on the gadget itself. Otherwise, people using the trace directly need to set different values. CLI: 4 or 6 vs Trace 2 or 10.

I agree even though I really do not like doing a lot of stuff on the container side.